mysql-test/suite/tianmu/t/issue852.test

```diff
@@ -0,0 +1,49 @@
+--source include/have_tianmu.inc
+
+USE test;
+
+CREATE TABLE IF NOT EXISTS `lineorder_flat` (
+  `LO_ORDERDATE` date NOT NULL COMMENT '',
+  `LO_ORDERKEY` int(11) NOT NULL COMMENT '',
+  `LO_LINENUMBER` tinyint(4) NOT NULL COMMENT '',
+   `LO_CUSTKEY` int(11) NOT NULL COMMENT '',
+  `LO_PARTKEY` int(11) NOT NULL COMMENT '',
+  `LO_SUPPKEY` int(11) NOT NULL COMMENT '',
+  `LO_ORDERPRIORITY` varchar(100) NOT NULL COMMENT '',
+  `LO_SHIPPRIORITY` tinyint(4) NOT NULL COMMENT '',
+  `LO_QUANTITY` tinyint(4) NOT NULL COMMENT '',
+  `LO_EXTENDEDPRICE` int(11) NOT NULL COMMENT '',
+  `LO_ORDTOTALPRICE` int(11) NOT NULL COMMENT '',
+  `LO_DISCOUNT` tinyint(4) NOT NULL COMMENT '',
+  `LO_REVENUE` int(11) NOT NULL COMMENT '',
+  `LO_SUPPLYCOST` int(11) NOT NULL COMMENT '',
+  `LO_TAX` tinyint(4) NOT NULL COMMENT '',
+  `LO_COMMITDATE` date NOT NULL COMMENT '',
+  `LO_SHIPMODE` varchar(100) NOT NULL COMMENT '',
+  `C_NAME` varchar(100) NOT NULL COMMENT '',
+  `C_ADDRESS` varchar(100) NOT NULL COMMENT '',
+  `C_CITY` varchar(100) NOT NULL COMMENT '',
+  `C_NATION` varchar(100) NOT NULL COMMENT '',
+  `C_REGION` varchar(100) NOT NULL COMMENT '',
+  `C_PHONE` varchar(100) NOT NULL COMMENT '',
+  `C_MKTSEGMENT` varchar(100) NOT NULL COMMENT '',
+  `S_NAME` varchar(100) NOT NULL COMMENT '',
+  `S_ADDRESS` varchar(100) NOT NULL COMMENT '',
+  `S_CITY` varchar(100) NOT NULL COMMENT '',
+  `S_NATION` varchar(100) NOT NULL COMMENT '',
+  `S_REGION` varchar(100) NOT NULL COMMENT '',
+  `S_PHONE` varchar(100) NOT NULL COMMENT '',
+  `P_NAME` varchar(100) NOT NULL COMMENT '',
+  `P_MFGR` varchar(100) NOT NULL COMMENT '',
+  `P_CATEGORY` varchar(100) NOT NULL COMMENT '',
+  `P_BRAND` varchar(100) NOT NULL COMMENT '',
+  `P_COLOR` varchar(100) NOT NULL COMMENT '',
+  `P_TYPE` varchar(100) NOT NULL COMMENT '',
+  `P_SIZE` tinyint(4) NOT NULL COMMENT '',
+  `P_CONTAINER` varchar(100) NOT NULL COMMENT ''
+) engine = tianmu ;
+
+--replace_result $MYSQL_TEST_DIR MYSQL_TEST_DIR
```

Comment:
The syntax of load data is listed as following:
```
LOAD DATA
    [LOW_PRIORITY | CONCURRENT] [LOCAL]
    INFILE 'file_name'
    [REPLACE | IGNORE]
    INTO TABLE tbl_name
    [PARTITION (partition_name [, partition_name] ...)]
    [CHARACTER SET charset_name]
    [{FIELDS | COLUMNS}
        [TERMINATED BY 'string']
        [[OPTIONALLY] ENCLOSED BY 'char']
        [ESCAPED BY 'char']
    ]
    [LINES
        [STARTING BY 'string']
        [TERMINATED BY 'string']
    ]
    [IGNORE number {LINES | ROWS}]
    [(col_name_or_user_var
        [, col_name_or_user_var] ...)]
    [SET col_name={expr | DEFAULT}
        [, col_name={expr | DEFAULT}] ...]
```

Reply:
Do we have a case for testing `load data` command and these options？

Reply:
https://dev.mysql.com/doc/refman/5.7/en/load-data.html
And here, we should pay more attention to `   FIELDS TERMINATED BY ',' ENCLOSED BY '"'
  LINES TERMINATED BY '\r\n'
  IGNORE 1 LINES` these options.

```
In certain cases, field- and line-handling options interact:

If LINES TERMINATED BY is an empty string and FIELDS TERMINATED BY is nonempty, lines are also terminated with FIELDS TERMINATED BY.

If the FIELDS TERMINATED BY and FIELDS ENCLOSED BY values are both empty (''), a fixed-row (nondelimited) format is used. With fixed-row format, no delimiters are used between fields (but you can still have a line terminator). Instead, column values are read and written using a field width wide enough to hold all values in the field. For [TINYINT](https://dev.mysql.com/doc/refman/5.7/en/integer-types.html), [SMALLINT](https://dev.mysql.com/doc/refman/5.7/en/integer-types.html), [MEDIUMINT](https://dev.mysql.com/doc/refman/5.7/en/integer-types.html), [INT](https://dev.mysql.com/doc/refman/5.7/en/integer-types.html), and [BIGINT](https://dev.mysql.com/doc/refman/5.7/en/integer-types.html), the field widths are 4, 6, 8, 11, and 20, respectively, no matter what the declared display width is.

LINES TERMINATED BY is still used to separate lines. If a line does not contain all fields, the rest of the columns are set to their default values. If you do not have a line terminator, you should set this to ''. In this case, the text file must contain all fields for each row.

Fixed-row format also affects handling of NULL values, as described later.

Note
Fixed-size format does not work if you are using a multibyte character set.
```

Reply:
SQL LOAD in tianmu is not so complete as mysql does, it has many limitations. We can improve it step by step. Or Maybe we can have special test for it to know how level it can support, and plan to which feature should be supportted first, It is not a small task.